### DISCUSSION TITLE
Redis.io and Tokio.rs

### DISCUSSION THREAD

**Discussion by QaisarAzeemi** [tokio-rs/tokio#3798] [General] [answered]
Hi,
I want to ask a very beginner level question. What actually redis is and why and where we use it?
same question is about Tokio. kindly tell me in a simple way i will be thankful.
following are the links of redis and tokio i am asking about.
https://redis.io/
https://tokio.rs/tokio/tutorial

waiting for the kind response of the community.
Thank you

**Comment by Darksonn** [comment, marked as answer]
Redis is a type of database or cache. Tokio is a library that lets you write networking applications in Rust. The tutorial implements a Redis server because Redis is a relatively simple networking application, so it is a good example.

**Reply by QaisarAzeemi** [reply]
Thanks a lot dear friend
